### PR TITLE
Updates release notes for 17.9.1

### PR DESCRIPTION
**Pull request by nkibler**

### CODE REVIEW

**Comment by etanshaul** [comment]
note about targeting 2017.3?

**Comment by etanshaul** [comment]
oh nm. I missed it

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,28 @@
 # Release notes
 This page documents production updates to Cloud Tools for IntelliJ. You can check this page for announcements about new or updated features, bug fixes, known issues, and deprecated functionality.
 
+## 17.9.1
+
+### Added
+  - Added the ability to change the name of the staged artifact for App Engine flexible deployments.
+    [1151](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1151)
```
Comment (etanshaul) [inline CHANGELOG.md]: is the link here mixed up with the link to the first bullet under 'changed'?

Comment (nkibler) [reply CHANGELOG.md]: I'm not sure what you mean; do you mean does the line-break cause an issue with formatting? Or do you mean are the links incorrectly swapped between this bullet and the next?

Comment (etanshaul) [reply CHANGELOG.md]: yea sorry - that the links are swapped between the two

Comment (nkibler) [reply CHANGELOG.md]: My original intention was that the addition of the field points to the tracking bug of the entire feature (#1151) and the default config change point to the PR that made it happen (#1611), but I can see how it's confusing. I can switch them, WDYT?

Comment (etanshaul) [reply CHANGELOG.md]: Seems to me like you should swap them and instead of pointing to the PR, point to the issue.

Comment (nkibler) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
 
+## 17.9.1
+
+### Added
+  - Added the ability to change the name of the staged artifact for App Engine flexible deployments.
+    [1610](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1610)
+
+### Changed
+  - App Engine flexible deployment configurations now default to deploy the artifact as-is, without
+    renaming to `target.jar` or `target.war`. [1151](https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/1151)
```
Comment (etanshaul) [inline CHANGELOG.md]: pull -> issue (I think github redirects it for you, but would be nice to fix)

Comment (nkibler) [reply CHANGELOG.md]: Whoops, done. Also added changes to `plugin.xml`.

**Pull request merged**